sefaria/helper/topic.py

```diff
@@ -1043,14 +1043,16 @@ def topic_change_category(topic_obj, new_category, old_category="", rebuild=Fals
     return topic_obj
 
 def update_topic_titles(topic_obj, data):
-    for lang in ['en', 'he']:
```

Comment:
This is feedback on the function in general and not on the changes made. `topic_obj` seems like it should be renamed to `topic` which seems more standard. It is preferable not to pass a parameter called `data` which is confusing to know what's in it. Instead, pass the contents of `data` as kwargs (i.e. `title`, `heTitle`, `altTitles`) and optional kwargs can default to `None` (it seems `altTitles` is optional). When calling `update_topic_titles()` you can call it using `**` like this `update_topic_titles(topic, **data)` which will pass the keys of `data` as kwargs.

Reply:
@nsantacruz [New test here](https://github.com/Sefaria/Sefaria-Project/pull/1691/commits/a005786221e3b63c984e988b7ff5f97e69564e09#diff-3eb576d5ed14f2bd2ce8aa0c80864a7f72f02c2ed94cbe4005879679af587534R94)
[Changed remove title here](https://github.com/Sefaria/Sefaria-Project/pull/1691/commits/a005786221e3b63c984e988b7ff5f97e69564e09#diff-ba861216d2529388aee5b8984af25f2eb0eacedee84c915c33d2952ead8ebb33R128) 
Changed the code back so that I [remove titles first](https://github.com/Sefaria/Sefaria-Project/pull/1691/commits/a005786221e3b63c984e988b7ff5f97e69564e09#diff-19364da9061bcb429f3f3c5f63aeab5541e461154bd59e61719b2cd68f631356R1048) because now it's possible

Reply:
This is great! Good job on the tests.